tests/markdown_code_obj.md

```diff
@@ -0,0 +1,9 @@
+```
+test markdown file for
+highlighing markdown codeblocks
+```
+
+```
+all code blocks
+should be highlighted
+```
```

Comment:
Please add a test including a code block with a language indicator.

Reply:
+1, I've seen both `py` and `python` language indicators in repos.

Reply:
Done, added support for language indicators and unit test for it.